RevenueCatUI/Views/TemplateBackgroundImageView.swift

```diff
@@ -54,6 +54,15 @@ struct TemplateBackgroundImageView: View {
             RemoteImage(url: url)
                 .blur(radius: 40)
                 .opacity(0.7)
+                .background {
+                    // Simulate dark materials in pre-watchOS 10.0
+                    // where `Material` isn't available.
+                    #if os(watchOS)
+                    if #unavailable(watchOS 10.0) {
+                        Color.black
+                    }
+                    #endif
```

Comment:
Just to confirm, in the else case, I guess this will be nil, so there won't be any background, correct?

Reply:
That's correct.